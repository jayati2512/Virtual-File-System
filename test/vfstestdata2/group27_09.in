createvfs demo 20
mountvfs demo
makedir /home/sub1/ sub2
makedir /etc/games/ movies
makedir /share/usr/ dummy
makedir /raj/rahhul/ravi/bhuvan/ragfavad/ dhiraj
deletedir /home/sub1/
deletedir /home/sub1/sub2/
addfile / abc.h /usr/include/syscall.h
addfile / def.txt /usr/include/blahblah.txt
addfile /share/usr/ ghi.h /usr/include/wait.h
unmountvfs demo
mountvfs demo
copyfile /share/usr/ghi.h /etc/games/
listdir / 1 abcd
addfile / abc.h /usr/include/wait.h
exportfile /share/usr/ghi.h /tmp/tmppEL2Rt
unmountvfs demo
mountvfs demo
movefile /share/usr/ghi.h /etc
unmountvfs demo
mountvfs demo
removefile /etc/ghi.txt
removefile /share/usr/ghi.h
unmountvfs demo
mountvfs demo
listdir / 1 /tmp/abcd
unmountvfs demo
