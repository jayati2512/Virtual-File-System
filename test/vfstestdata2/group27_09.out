createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
deletedir_SUCCESS
addfile_SUCCESS
addfile_FAILURE UNABLE_TO_CREATE_NEWFILE
addfile_SUCCESS
unmountvfs_SUCCESS
mountvfs_SUCCESS
copyfile_FAILURE CANNOT_FIND_DESTINATIONPATH
listdir_SUCCESS
addfile_FAILURE FILE_ALREADY_EXISTS
exportfile_FAILURE CANNOT_CREATE_OUTPUTFILE
unmountvfs_SUCCESS
mountvfs_SUCCESS
movefile_FAILURE CANNOT_FIND_DESTINATION_PATH
unmountvfs_SUCCESS
mountvfs_SUCCESS
removefile_FAILURE CANNOT_FIND_SPECIFIED_FILE
removefile_SUCCESS
unmountvfs_SUCCESS
mountvfs_SUCCESS
listdir_SUCCESS
unmountvfs_SUCCESS
